From ea3430b0338bb98a91e54bde7b3a75c4c396a430 Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Mon, 22 Jan 2018 10:09:59 +0800
Subject: [PATCH 3225/5242] MLK-17438 arm64: dts: add emvsim0 device node in
 register address order

commit  ed06d988e7806ad92b3d20ef13305502ad7996b8 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add emvsim0 device node in register address order

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
Acked-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |   22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 6716e84..54a2536 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -1416,17 +1416,6 @@
 		};
 	};
 
-	emvsim0: sim0@5a0d0000 {
-		compatible = "fsl,imx8-emvsim";
-		reg = <0x0 0x5a0d0000 0x0 0x10000>;
-		interrupts = <GIC_SPI 230 IRQ_TYPE_LEVEL_HIGH>;
-		clocks = <&clk IMX8QM_EMVSIM0_CLK>,
-			 <&clk IMX8QM_EMVSIM0_IPG_CLK>;
-		clock-names = "sim", "ipg";
-		power-domains = <&pd_ldo1_sim>;
-		status = "disabled";
-	};
-
 	hdmi:hdmi@56268000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
@@ -2570,6 +2559,17 @@
 		status = "disabled";
 	};
 
+	emvsim0: sim0@5a0d0000 {
+		compatible = "fsl,imx8-emvsim";
+		reg = <0x0 0x5a0d0000 0x0 0x10000>;
+		interrupts = <GIC_SPI 230 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&clk IMX8QM_EMVSIM0_CLK>,
+			 <&clk IMX8QM_EMVSIM0_IPG_CLK>;
+		clock-names = "sim", "ipg";
+		power-domains = <&pd_ldo1_sim>;
+		status = "disabled";
+	};
+
 	edma0: dma-controller@5a1f0000 {
 		compatible = "fsl,imx8qm-edma";
 		reg = <0x0 0x5a2c0000 0x0 0x10000>, /* channel12 UART0 rx */
-- 
1.7.9.5

